romanisim/bandpass.py

```diff
@@ -129,3 +133,26 @@ def get_abflux(bandpass):
         abflux = compute_abflux()
         get_abflux.abflux = abflux
     return abflux[bandpass]
+
+def compute_count_rate(thru=None, bandpass=None, filename=None, effarea=None):
```

Comment:
Can we rename thru -> spectrum or something?  thru sounds like throughput to me but that's not what that argument is?

IMO thru and bandpass should be required arguments and not keywords that default to None?

Reply:
Renamed to flux and both are set to required.